docs/src/content/components/errors.mdx

```diff
@@ -21,16 +16,16 @@ import {
 
 ## Examples
 
-### 403 Forbidden
+### 403 Forbidden (size=small)
```

Comment:
I would keep the 403 example more consistent with the existing examples in the gallery so far, keeping default size.  Then, I'd make a new and separate example showing that size can be adjusted. This so-as to show that:

- 403 acts like the rest of the pre-built error pages, minus copy
- Size small is applicable to any and all of the error pages